openapi.yaml

```diff
@@ -0,0 +1,69 @@
+swagger: '2.0'
+info:
+  version: '1.0.0'
+  title: Fulcio
+schemes: [http, https]
+host: fulcio.sigstore.dev
+basePath: /v1
```

Comment:
I'd prepend `/api/` to this so if you wanted to have a webpage with instructions/etc you can safely do that.